services/network/test_doubles/ConnectionStub.hpp

```diff
@@ -30,6 +30,7 @@ namespace services
         void SimulateDataReceived(infra::ConstByteRange data);
 
         std::vector<uint8_t> sentData;
+        std::size_t maxSendStreamSize = 1024;
```

Comment:
```suggestion
        const std::size_t maxSendStreamSize{ 1024 };
```

Reply:
The intention was that tests can change this parameter